src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeClientStream.Unix.cs

```diff
@@ -28,7 +28,7 @@ private bool TryConnect(int timeout, CancellationToken cancellationToken)
             SafePipeHandle? clientHandle = null;
             try
             {
-                socket.Connect(new UnixDomainSocketEndPoint(_normalizedPipePath));
+                socket.Connect(new UnixDomainSocketEndPoint(_normalizedPipePath!));
```

Comment:
Assume it is intended

Reply:
Yes, this change is needed. I logged https://github.com/dotnet/runtime/issues/32760 to track this scenario.